be/src/storage/base_and_cumulative_compaction_policy.cpp

```diff
@@ -158,6 +158,9 @@ void BaseAndCumulativeCompactionPolicy::_pick_base_rowsets(std::vector<RowsetSha
     size_t rowsets_compaction_score = 0;
     // add the base rowset to input_rowsets
     Rowset* base_rowset = *_compaction_context->rowset_levels[2].begin();
+    if (base_rowset == nullptr) {
```

Comment:
in which situation there is no base rowset?

Reply:
The clone of tablet may lead no base rowset.